terraform/aks/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "app_name_suffix" { default = null }
+
+variable "postgres_version" { default = 11 }
+
+variable "app_name" { default = null }
```

Comment:
I believe app_name is only used because of review apps. This can be removed to simplify as we don't have review apps here